src/Mono.Android/Test/Mono.Android-Tests.projitems

```diff
@@ -4,7 +4,9 @@
     <UnitTestApk Include="$(OutputPath)Mono.Android_Tests-Signed.apk">
       <Package>Mono.Android_Tests</Package>
       <InstrumentationType>xamarin.android.runtimetests.TestInstrumentation</InstrumentationType>
-      <ResultsPath>$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
+      <ResultsPath Condition=" '$(Configuration)' != 'Release' " >$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
```

Comment:
Plus I wasn't sure whether other parts (Jenkins, wrench?) use the `TestResult-Mono.Android_Tests.xml` filename, so the original PR version keeps the original filename.